src/appleseed.python/bindblenderexport.cpp

```diff
@@ -326,8 +453,18 @@ void export_mesh_blender80_pose(
     }
 }
 
-void bind_blender_mesh_converter()
+void bind_blender_export()
 {
+    bpy::class_<BlTransformLibrary, auto_release_ptr<BlTransformLibrary>, boost::noncopyable>("BlTransformLibrary", bpy::no_init)
+        .def("__init__", bpy::make_constructor(create_bl_transform_library))
+        .def("get_size", &BlTransformLibrary::get_size)
```

Comment:
If you want to be able to use `len()` in Python, you need to change `get_size` to `__len__`. I think that would be more natural.

You can also rename the C++ method to `size()`, that's how all such methods are called in the standard library.